### PR TITLE
Refactor for easier readability

### PR DESCRIPTION
**Pull request by waymobetta**
## Description
Refactor for easier readability

## Related Issue Or Context

Closes: N/A

## How Has This Been Tested? Testing details.
N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Checklist:
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [x] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [x] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: docs/local.md [hunk 1]
```diff
@@ -1,23 +1,71 @@
 # Running Locally
 ## Prerequisites
 
-- Docker, docker-compose
+- Docker (**Note:** docker-compose now comes preinstalled with Docker)
```
Comment (ansermino) [inline docs/local.md:4]: This only applies to Windows and OSX 😢  https://docs.docker.com/compose/install/

Will need to update cmds as well.

**Review by ansermino — COMMENTED** [review]
Some fantastic improvements here! 👏 🎉 

One small thing to fix.

File: docs/local.md (hunk 1 above)
Comment (waymobetta) [reply docs/local.md]: Good catch @ansermino ..but also 😢

**Comment by P1sar** [comment]
Since docker-compose-e2e.yml is using 1.3.0 substrate chain version it is required to update Chainbridge config template from section https://github.com/ChainSafe/chainbridge-docs/blob/main/docs/local.md#running-a-relayer by adding 
"useExtendedCall": "true" to the "opts" : {} of the ETH config

File: docs/local.md [hunk 2]
```diff
@@ -3,21 +3,69 @@
...
+7. [`Register Resources Substrate`](#register-resources-substrate)
+8. [`Whitelist Chains`](#whitelist-chains)
+9. [`Run Relayer`](#run-relayer)
+10. [`Fungible Transfers`](#fungible-transfers)
+11. [`Non-Fungible Transfers`](#non-fungible-transfers)
 
-The easiest way to get started is to use the docker-compose file found here: [https://gist.github.com/ansermino/f1571bb354d2007b26dce53d52dbca75](https://gist.github.com/ansermino/f1571bb354d2007b26dce53d52dbca75). This will start a geth instance and an instance of chainbridge-substrate-chain:
+## Start Local Chains
+
+The easiest way to get started is to use the docker-compose file found [here](https://github.com/ChainSafe/ChainBridge/blob/main/docker-compose-e2e.yml). This will start two geth instances and an instance of chainbridge-substrate-chain:
```
Comment (ansermino) [inline docs/local.md:23]: Lets remove the link and add the config file directly.

Comment (waymobetta) [reply docs/local.md:23]: So we should use your original [docker-compose gist](https://gist.github.com/ansermino/f1571bb354d2007b26dce53d52dbca75) and stick with:
- ChainBridge v1.1.0
- chainsafe/chainbridge-substrate-chain:v1.1.0

OR, we should use [docker-compose-e2e.yml](https://github.com/ChainSafe/ChainBridge/blob/main/docker-compose-e2e.yml) and update to:
- ChainBridge v1.1.1
- chainsafe/chainbridge-substrate-chain:v1.3.0
- `opts:{"useExtendedCall": "true"}`

**Review by P1sar — APPROVED** [review]

File: docs/local.md [hunk 3]
```diff
@@ -240,11 +259,13 @@ Sample `config.json`:
...
     {
       "name": "sub",
-      "type": "Substrate",
+      "type": "substrate",
       "id": "1",
       "endpoint": "ws://localhost:9944",
       "from": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
-      "opts": {}
+      "opts": {
+          "useExtendedCall":"true",
```
Comment (kevingzhang) [inline docs/local.md:267]: Please remove the last comma, this causes a loading config error. JSON has weird format requirements but we have to follow.

Comment (waymobetta) [reply docs/local.md:267]: Thanks for finding this and letting us know. We'll submit a fix asap

**Pull request merged**